recipe/meta.yaml

```diff
@@ -10,30 +10,43 @@ source:
   sha256: 3be9ca9d812e760969419e7288fef4d6700e7aa01d303aad4e56cd4ecb4f11ed
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record=record.txt
   # This packages uses libhdfs3, which is primarily used with HDFS on Linux.
-  skip: True  # [win]
+  # There is no libhdfs3 >=2.3 and its dependencies on s390x
+  skip: True  # [win or osx or s390x]
 
 requirements:
   host:
     - python
+    - pip
     - setuptools
-
+    - wheel
```

Comment:
Just a suggestion, but pip already depends on wheel, so unless we use wheel directly, it seems we don't need to mention it as a dependency.

Reply:
Kai recommends explicitly use`wheel` in `host`. And his [crender](https://github.com/katietz/crender) tool recommends the same

Reply:
Hm, ok.